### DISCUSSION TITLE
Overview of data using visualisation

### DISCUSSION THREAD

**Discussion by zain-sohail** [momentoscope/hextof-processor#101] [Ideas]
As suggested by Lukas and Dima multiple times, we should create some plots for overview in python with functions:

- plotTLD
- plotIR
- plotGMD

**Comment by kutnyakhov** [comment]
![Screenshot_panelOverview](https://user-images.githubusercontent.com/38926659/140079101-c0b12f7e-af16-4caa-9156-52ce639853d9.png)
